app/modules/ads/index.js

```diff
@@ -86,6 +86,13 @@ class Ads {
     this.showAds = this.showAds && mediaWikiAdsContext.opts.pageType !== 'no_ads';
 
     adsSetup.configure(mediaWikiAdsContext, instantGlobals, isOptedIn);
+
+    if (context.get('options.disableAdStack')) {
+      pageTracker.trackProp('adengine', 'off');
+      this.trackLabrador();
+      return;
```

Comment:
It breaks FV. Video does not play. I'm not sure, but probably we need to start onReadyCallbacks/isLoaded. What about `this.showAds` flag? Does switching this flag solve the issue?